spec/granite/query/assemblers/mysql_spec.cr

```diff
@@ -3,13 +3,19 @@ require "../spec_helper"
 {% if env("CURRENT_ADAPTER").id == "mysql" %}
   describe Granite::Query::Assembler::Mysql(Model) do
     context "count" do
-      it "adds group_by fields for where/count queries" do
-        sql = "select count(*) from table where name = ? group by name"
-        builder.where(name: "bob").count.raw_sql.should match ignore_whitespace sql
+			it "counts for where/count queries" do
+				sql = "select count(*) from table where name = ?"
+				builder.where(name: "bob").assembler.count.raw_sql.should match ignore_whitespace sql
```

Comment:
You shouldn't add `assembler` here.  It's not intuitive and it should work without it.

Reply:
i need to use assembler for receive count's raw sql
if i use only `raw_sql` its giving select's raw sql

Reply:
hmm, something doesn't sound right if that is the case.  I would expect `builder.where(name: "bob).count.raw_sql` to return the correct sql.

Reply:
`builder` have this code so `where(name: "bob")` `return builder`
```
  def raw_sql
    assembler.select.raw_sql
  end
```
when we call `select` or `count` or `delete` its `return executer` and its have `raw_sql` their sql not same